src/components/Banner/Banner.tsx

```diff
@@ -44,6 +44,11 @@ export interface Props {
 
 export default class Banner extends React.PureComponent<Props, never> {
   static contextTypes = contentContextTypes;
+  private wrapper = React.createRef<HTMLDivElement>();
+
+  public focus() {
+    this.wrapper.current && this.wrapper.current.focus();
```

Comment:
Normally this would be an early return but that leaves untestable code so this is an alternative.